spec/integration/delete_user_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
```

Comment:
We should probably DRY these tests with some test helpers - lets add it to the list if we get time in a future sprint